doc/misc/npm-scripts.md

```diff
@@ -119,6 +119,8 @@ executing the scripts.  So, if your package.json has this:
 then you could run `npm start` to execute the `bar` script, which is
 exported into the `node_modules/.bin` directory on `npm install`.
 
+This is especially important for proper Windows support.
```

Comment:
Needs more context. What aspect of this is important for Windows support? Also, "proper" is a content-free word in this context.